plugins/repository-s3/src/main/java/org/opensearch/repositories/s3/S3RepositoryPlugin.java

```diff
@@ -234,6 +236,7 @@ public Collection<Object> createComponents(
 
         this.lowTransferQConsumerService = threadPool.executor(LOW_TRANSFER_QUEUE_CONSUMER);
         this.normalTransferQConsumerService = threadPool.executor(NORMAL_TRANSFER_QUEUE_CONSUMER);
+        this.scheduler = threadPool.scheduler();
```

Comment:
Is this needed?

Reply:
no, sorry. This is a leftover of a failed attempt to re-use elastic scheduler. Could possibly be a good idea but turned out to be a bit more involved than I anticipated and did not feel confident enough to pursue on this route.